tests/test_models/test_heads/test_cliff_head.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import torch
+
+from mmhuman3d.models.heads.builder import CliffHead
+
+
+def test_cliff_head():
```

Comment:
Might add a test for CLIFFMeshEstimator if we decide to write a individual file for CLIFF.